rustls-mbedpki-provider/src/tests_common.rs

```diff
@@ -12,24 +12,22 @@ use core::{
     ops::{Deref, DerefMut},
 };
 
-use pki_types::{CertificateDer, PrivateKeyDer, PrivatePkcs8KeyDer, UnixTime};
+use pki_types::{CertificateDer, PrivateKeyDer, PrivatePkcs8KeyDer, ServerName, UnixTime};
```

Comment:
```suggestion
use pki_types::{CertificateDer, PrivateKeyDer, ServerName, UnixTime};
```
Remove unused type, see  comment below